pkgs/applications/science/electronics/kicad/base.nix

```diff
@@ -46,29 +49,22 @@ assert ngspiceSupport -> libngspice != null;
 
 with lib;
 let
-  versionConfig = versions.${baseName};
-
   # oce on aarch64 fails a test
   withOCE = oceSupport && !stdenv.isAarch64;
```

Comment:
This is not part of the PR, I know. This should be an assertion and not a silent disable.